pkg/cmd/projectconfig/import.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package projectconfig
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/charmbracelet/huh"
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	"github.com/daytonaio/daytona/pkg/apiclient"
+	"github.com/daytonaio/daytona/pkg/views"
+	views_util "github.com/daytonaio/daytona/pkg/views/util"
+	"github.com/daytonaio/daytona/pkg/views/workspace/create"
+	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
+	"github.com/spf13/cobra"
+)
+
+var (
+	projectConfigList []apiclient.ProjectConfig
```

Comment:
Why is this a "global" variable?

Reply:
This is global because `ListProjectConfigs` is called only once, and I'm appending the imported project configs to the array. This helps when a user imports an array of project configs that might have same names. Making it global will avoid importing those PCs.

Reply:
This can also be solved with pointers. Should I do it that way?

Reply:
> This can also be solved with pointers. Should I do it that way?

Yeah, you can pass the array around as an argument